go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/DataDog/zstd v0.0.0-20180321192404-aebefd9fcb99
 	github.com/apache/beam v0.0.0-20181024230410-cdd3458380a6
 	github.com/beevik/etree v0.0.0-20180609112216-9d7e8feddccb
+	github.com/ghodss/yaml v1.0.0
```

Comment:
This should be locked down to the commit used by Bazel.

Reply:
:+1: